lib/galaxy/tools/parameters/basic.py

```diff
@@ -21,25 +20,24 @@
 from ..parameters import dynamic_options
 from .dataset_matcher import DatasetMatcher
 from .dataset_matcher import DatasetCollectionMatcher
-# For BaseURLToolParameter
 from galaxy.web import url_for
 from galaxy.util.dictifiable import Dictifiable
 import galaxy.model
 from galaxy.util.bunch import Bunch
 
-log = logging.getLogger(__name__)
+log = logging.getLogger( __name__ )
 
 workflow_building_modes = Bunch( DISABLED=False, ENABLED=True, USE_HISTORY=1 )
 
-WORKFLOW_PARAMETER_REGULAR_EXPRESSION = re.compile( '''\$\{.+?\}''' )
+workflow_parameter_regular_expression = re.compile( '''\$\{.+?\}''' )
```

Comment:
I'd switch this back - the PEP-8 standard is pretty clear and we don't have any stated modifications to it with respect to constant. https://www.python.org/dev/peps/pep-0008/#constants